kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -138,6 +138,14 @@ window.ExerciseView = Backbone.View.extend({
 
         // Catch the "next question" button click event -- needs to be explicit (not in "events")
         this.$("#next-question-button").click(this.next_question_clicked);
+        
+        // Workaround so that pressing enter on perseus exercises submits the answer 
+        var selector = $(".perseus-input");
```

Comment:
Also, I don't think enter is working on non-perseus exercises too, so may be worth double checking this on addition_1 too.